_data/menu.yml

```
@@ -4,10 +4,44 @@ options:
     url: quickstart
 
   - title: Getting Started
-    url: docs/getting-started
+    url: docs
 
   - title: Introduction to Effekt
-    url: docs
+    url: docs/introduction
+
+  - title: Language Tour
+    url: docs/tutorial
+    nested_options:
+      - title: Hello World
+        url: docs/tutorial/hello-world
+      - title: Values
+        url: docs/tutorial/values
+      - title: Variables
+        url: docs/tutorial/variables
+      - title: Computations
+        url: docs/tutorial/computation
+      - title: Records
+        url: docs/tutorial/records
+      - title: Datatypes
+        url: docs/tutorial/adts
+      - title: Loops
+        url: docs/tutorial/loops
+      - title: Functions
+        url: docs/tutorial/functions
+      - title: Effects
+        url: docs/tutorial/effects
+      - title: Objects
+        url: docs/tutorial/objects
+      - title: Captures
+        url: docs/tutorial/captures
+      - title: Namespaces
+        url: docs/tutorial/namespaces
+      - title: Regions
+        url: docs/tutorial/regions
+      - title: IO
```

```suggestion
      - title: File IO
```
since we are currently only discusssing this

---

In the Effekt repo this file has the title `Asynchronous IO (Input-Output)`. I'd just leave it like this to not have two different names. Also, we will probably expand this section with other parts of IO in the future.